src/index.ts

```diff
@@ -229,10 +229,15 @@ class Marian {
     try {
       // TODO: include taxonomy url in verifyEnvVars after it has been released
       taxonomy = await this.fetchTaxonomy(process.env.TAXONOMY_URL!);
-      const atlasAdminRes = await this.atlasAdmin.patchSearchIndex(taxonomy);
-      const loadRes = await this.index.load(taxonomy);
+      await this.atlasAdmin.updateSynonyms();
+
+      const [atlasAdminRes, loadRes] = await Promise.all([
```

Comment:
Just wondering if responses from these two need to be saved? Looks like they're unused

Reply:
I don't believe they are used now, I just modified the [existing code](https://github.com/mongodb/docs-search-transport/blob/dc0de2e5255cf959ffbcb22551b0aab6fea4c26c/src/index.ts#L233) to do these requests in parallel while keeping the results stored as a variable. Not sure if they are planned on being used in the future, but I kept them in just in case. Would you prefer if I remove them now and just add them in later as needed?

Reply:
My general preference is to not have them if we don't use them/plan on using them in order to avoid any potential confusion. But @seungpark would probably have more context on this!